### PR TITLE
Labs branch

### PR DESCRIPTION
**Pull request by mightynimble**

### CODE REVIEW

**Comment by mightynimble** [comment]
I will work on reverting ```find().trigger('click')``` back to ```.click``` first.

File: spec/features/granules/granule_filters_spec.rb [hunk 1]
```diff
@@ -245,65 +246,67 @@
...
-        it "includes the excluded granules in the list" do
-          expect(page).to have_css('#granule-list .panel-list-item', count: 20)
-        end
-
-        it "updates the granule hits count" do
-          expect(page).to have_content("Showing 20 of #{before_granule_count} matching granules")
-        end
-      end
-    end
+    # JS: New layout may make these tests invalid
```
Comment (mreese84) [inline spec/features/granules/granule_filters_spec.rb]: Granules can still be excluded individually. Do we still have tests to cover that? Or do these just need to be re-written?

Comment (mightynimble) [inline spec/features/granules/granule_filters_spec.rb]: Yeah, that is one of my questions in the email. I don't know why these tests are excluded.

**Comment by mreese84** [comment]
Closing this PR in favor of the new, less confusing, #389. @jsiarto, make sure you've got all of the comments @mightynimble left documented or addressed.

**Pull request closed without merging**